src/debian/12/gcc14/amd64/Dockerfile

```
@@ -29,3 +30,17 @@ ENV LANG=en_US.utf8
 # In the runtime repo, we (by design) give precedence to suffixed compilers before selecting unsuffixed one in PATH.
 RUN ln -s $(command -v gcc) /usr/bin/gcc-14 && \
     ln -s $(command -v g++) /usr/bin/g++-14
+
+# Install powershell. This is in the debpkg/amd64 docker file as it may not be required in the arm64
+# images, and is not actually supported over there.
+RUN apt-get update && \
+    apt-get install -y \
+      apt-transport-https \
+      software-properties-common \
+      curl && \
+    curl -sL https://packages.microsoft.com/config/debian/11/packages-microsoft-prod.deb -o packages-microsoft-prod.deb && \
```

Shouldn't this be 12?
```suggestion
    curl -sL https://packages.microsoft.com/config/debian/12/packages-microsoft-prod.deb -o packages-microsoft-prod.deb && \
```

---

Or is this due to what you mentioned in chat?
> the newest version of the azure-cli availble doesn't appear to have the correct parameters to support the AzureCLI task

---

powershell isn't actually available on the debian 12 feed for whatever reason

---

Updated comment